lib/puppetserver/ca/utils/http_client.rb

```diff
@@ -195,7 +195,13 @@ def self.check_server_online(settings, logger)
           rescue Puppetserver::Ca::ConnectionFailed => e
             if e.wrapped.is_a? Errno::ECONNREFUSED
               return false
+            elsif suppress_error
+              logger.inform("Could not determine that Puppet Server is offline, " \
+                "connection check failed with error: #{e.wrapped}.")
+              return false
             else
+              logger.inform("An error has happened. Please check if Puppet Server is running.\n" \
+                "If '--ca-client' flag was used, try again with the '--force' flag to avoid the issue.")
```

Comment:
Okay I changed my mind, I think this _should_ go in the outer class, so we can know for sure if they used `ca-client` or not.
 As for wording, let's be more specific as to the issue and the remedy, something like:
"Could not determine if Puppet Server is online. Please ensure it is offline, then run this command again with `--force`."